pandas/core/groupby/generic.py

```diff
@@ -1653,7 +1649,7 @@ def _gotitem(self, key, ndim: int, subset=None):
 
         raise AssertionError("invalid ndim for _gotitem")
 
-    def _wrap_frame_output(self, result, obj: DataFrame) -> DataFrame:
+    def _wrap_frame_output(self, result: dict, obj: DataFrame) -> DataFrame:
```

Comment:
this is dict[base.OutputKey, FrameOrSeriesUnion] i think, but certainly can be next pass. we allow raw dict in typing?

Reply:
looks like it is `dict[Hashable, NDFrame | np.ndarray]`, but ill want to look at this more closely later